feature-utils/poly-look/src/css/design-system-tokens/spacing.css

```diff
@@ -13,7 +15,15 @@ Potential use for future "Figma tokens" variables. Currently used in Banner and
   --pl-space-9x: calc(var(--pl-space-1x) * 9);
   --pl-space-10x: calc(var(--pl-space-1x) * 10);
 
+  /* Component tokens */
```

Comment:
Is there some best practice here to indicate hierarchy? At first sight, it could be an empty section or a header with higher level than the one below. Indentation, markdown, could all help. There are some indications [here](https://perishablepress.com/obsessive-css-code-formatting-organization-comments-and-signatures/), for instance, but not sure how mainsream they are.

Reply:
Honestly I don't know. I've just [copied ](https://wiki.polypoly.eu/display/POLYPOD/Spacing#Spacing-Tokens)what the sculptors did.